src/mdacli/logger.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+#
+# Copyright (c) 2021 Authors and contributors
+#
+# Released under the GNU Public Licence, v2 or any higher version
+# SPDX-License-Identifier: GPL-2.0-or-later
+"""Logging."""
+
+import contextlib
+import logging
+from logging.handlers import RotatingFileHandler
+
+from mdacli.colors import Emphasise
+
+
+DEBUGFORMATTER = '[{levelname}] {filename}:{name}:{funcName}:{lineno}:'
+DEBUGFORMATTER += '{message}'
+"""Debug file formatter."""
+
+INFOFORMATTER = '{message}'
+"""Log file and stream output formatter."""
+
+logger = logging.getLogger(__name__)
```

Comment:
since you have the same in `__init__`, why here again? May be there is a reason but I am not seeing it.

Reply:
There is. We need it further down in L64